src/components/Settings.css

```diff
@@ -38,3 +38,12 @@
 .Settings-input-error {
   border-color: #cc0000;
 }
+
+.settings-dropdown {
```

Comment:
Since we're pattern-matching here, `.settings-dropdown` should be `.Settings-dropdown` (and obviously changed in associated JSX)

Reply:
Done